.azure-pipelines/client.yml

```diff
@@ -10,24 +10,36 @@ jobs:
   pool:
     vmImage: 'ubuntu-16.04'
 
+  strategy:
+    matrix:
+      Java 8:
+        artifactDir: 'java-8'
```

Comment:
This is going to cause issues in our release pipelines. We should avoid dropping both shipping and non-shipping assets into the same artifact.